packages/middleware-flexible-checksums/src/constants.ts

```diff
@@ -74,8 +74,3 @@ export enum ChecksumLocation {
  * @internal
  */
 export const DEFAULT_CHECKSUM_ALGORITHM = ChecksumAlgorithm.CRC32;
-
-/**
- * @internal
- */
-export const S3_EXPRESS_DEFAULT_CHECKSUM_ALGORITHM = ChecksumAlgorithm.CRC32;
```

Comment:
is it possible S3 will change the "default algorithm" again while intending S3 Express to stay on CRC32?